_sass/custom.scss

```diff
@@ -37,6 +37,13 @@
       margin-bottom: 0;
     }
   }
+
+  blockquote {
+    p, ul, ol {
+      padding: 0.1rem;
```

Comment:
oh man this has bugged me for ages. I scanned a few old articles to see if it still looks ok- https://deploy-preview-314--interrupt.netlify.app/authors/noah/ selfishly, and I think it's good!

Reply:
Just quickly scanned all articles that have blockquotes and I think everyone is using them more for notes than really "quotes" so switching to non-italic and smaller margins probably works for everyone.

The best (and only example) I found of a complex multiline note is this. I have improved a bit more for it:  https://interrupt.memfault.com/blog/installing-gdb

Before:
![Screen Shot 2023-05-08 at 4 18 57 PM](https://user-images.githubusercontent.com/779005/236957621-e60adb77-b335-4bef-8a1f-3136a260ae2d.png)

After:
![image](https://user-images.githubusercontent.com/779005/236958232-c951bdc1-4d1e-4985-8061-edbb005efebb.png)




I have not found example of multilines quotes. Probably because they looked really bad before ;)